app/constants/events.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  BATCH_PROCESSED: 'uk.gov.defra.ffc.pay.batch.processed.dps',
+  BATCH_REJECTED: 'uk.gov.defra.ffc.pay.warning.batch.rejected',
+  BATCH_QUARANTINED: 'uk.gov.defra.ffc.pay.warning.batch.quarantined',
+  PAYMENT_EXTRACTED: 'uk.gov.defra.ffc.pay.payment.extracted',
+  PAYMENT_REJECTED: 'uk.gov.defra.ffc.pay.warning.payment.rejected'
```

Comment:
Probably need different events for these deposits as if we use payment, they will end up in the payment event store where it will fail validation as the `data` element is not a payment request.

We probably should add a seperate store for security deposits.  I'd do that as a seperate ticket and for now leave the extracted and rejected events out of this PR